packages/components/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -12,30 +14,69 @@ type ThemeProviderProps = {
      * The theme to apply
      * @default "light"
      * */
-    theme: AvailableTheme;
+    theme?: AvailableTheme;
     /**
      * Direction to apply to the theme
      * @default "ltr"
      */
-    dir: 'ltr' | 'rtl';
+    dir?: 'ltr' | 'rtl';
+    /**
+     * Translations object to use for component strings.
+     * Import from '@frontify/fondue/components/locales' or provide your own.
+     * Must be a complete Translations object.
+     * @default English translations
+     * @example
+     * ```tsx
+     * import { de } from '@frontify/fondue/components/locales';
+     * <ThemeProvider translations={de}>...</ThemeProvider>
+     * ```
+     */
+    translations?: Translations;
     /**
      * Change the default rendered element for the one passed as a child, merging their props and behavior.
      * @default false
      */
     asChild?: boolean;
 };
 
-export const ThemeContext = createContext<Pick<ThemeProviderProps, 'theme' | 'dir'>>({ theme: 'light', dir: 'ltr' });
+type ThemeContextValue = {
+    theme: AvailableTheme;
+    dir: 'ltr' | 'rtl';
+    translations: Translations;
+};
+
+export const ThemeContext = createContext<ThemeContextValue>({
+    theme: 'light',
+    dir: 'ltr',
+    translations: en,
+});
 ThemeContext.displayName = 'ThemeContext';
 
 export const useFondueTheme = () => {
     return useContext(ThemeContext);
 };
 
-export const ThemeProvider = ({ children, theme = 'light', dir = 'ltr', asChild = false }: ThemeProviderProps) => {
+export const ThemeProvider = ({
+    children,
+    theme = 'light',
+    dir = 'ltr',
+    translations: customTranslations,
```

Comment:
```suggestion
    translations: customTranslations = en,
```
so we can remove the `|| en`?